backend/src/seeds/graphql/queries.ts

```diff
@@ -156,7 +156,7 @@ export const getCommunities = gql`
       id
       foreign
       publicKey
-      url
+      endPoint
```

Comment:
if you cut-off the `apiVersion` from `url` and rename it now to `endpoint`, you need at least to return the apiVersion not to miss some details.

Reply:
right